argusclient/client.py

```diff
@@ -139,17 +139,19 @@ def __init__(self, argus):
 
 
 class BaseModelServiceClient(object):
-    def __init__(self, argus, get_all_path=None):
+    def __init__(self, argus, get_all_path=None, get_all_path_params=None):
```

Comment:
How about calling the new param just `get_all_param`? I think it conveys the same meaning by being shorter. Read it as "path and params for get_all operation".
```suggestion
    def __init__(self, argus, get_all_path=None, get_all_params=None):
```

Reply:
It was actually get_all_params before and you suggested changing it to get_all_path_params. I think it's fine either way though. Changed it back.

Reply:
I am sorry about that, I think I made that comment when I didn't fully understand the purpose or usage :)